### DISCUSSION TITLE
Link de 1era sesión BackEnd

### DISCUSSION THREAD

**Discussion by oscararcee** [Launch-X-Latam/MisionFrontEnd#530] [Team Posts]
Hola compañeros, buenos días.

¿Alguien sería tan amable de compartirme la liga para poder ver la primera sesión de BackEnd por favor? Ya la busqué, pero no le encuentro y quiero ponerme al corriente para no atrasarme.

Saludos.

**Comment by jjmillar** [comment]
El stream esta en el Teams del Backend en GENERAL. De igual manera te dejo el link :D

https://teams.microsoft.com/l/meetup-join/19%3ameeting_MmQ0YTUwMWEtZWQ5ZS00MmI3LTlkYjUtNTQ5NmIxZTQ0YjIz%40thread.v2/0?context=%7b%22Tid%22%3a%224ae54b05-b77e-4224-aef1-8661422e0816%22%2c%22Oid%22%3a%22c3d76ba1-72de-4bf4-be64-d1c6a6bd4143%22%2c%22IsBroadcastMeeting%22%3atrue%7d
